src/rest/server/router.go

```diff
@@ -23,19 +23,63 @@ import (
 	"net/http"
 	"strings"
 	"time"
-
+	"bytes"
 	"github.com/golang/glog"
 	"github.com/gorilla/mux"
+	"fmt"
 )
 
 // Root directory for UI files
 var swaggerUIDir = "./ui"
-var UserAuth struct {
-	User bool
-	Cert bool
-	Jwt bool
+type UserAuth map[string]bool
+
+var ClientAuth = UserAuth{"user": false, "cert": false, "jwt": false}
+
+func (i UserAuth) String() string {
+    b := new(bytes.Buffer)
+    for key, value := range i {
+        if value {
+                fmt.Fprintf(b, "%s ", key)
```

Comment:
nitpick: indentation consistency here and throughout

Reply:
I have noticed a mix of tabs and spaces in files.. we may need to run a lint on all of the source files.